Owncloud iOs Client/Utils/FileNameUtils.m

```diff
@@ -269,15 +269,44 @@ + (BOOL) isForbiddenCharactersInFileName:(NSString*)fileName withForbiddenCharac
 }
 
 
-+ (BOOL)isURLWithSamlFragment:(NSString*)urlString{
++ (BOOL) isURLWithSamlFragment:(NSHTTPURLResponse *)response {
     
     // NSString *samlFragment1 = @"AuthnEngine";
+    
+    NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *) response;
+    NSDictionary *dict = [httpResponse allHeaderFields];
+    //Server path of redirected server
+    NSString *responseURLString = [dict objectForKey:@"Location"];
+
+    responseURLString = [responseURLString lowercaseString];
+    
+    if (responseURLString) {
+        for (NSString* samlFragment in kSAMLFragmentArray) {
+            if ([responseURLString rangeOfString:samlFragment options:NSCaseInsensitiveSearch].location != NSNotFound) {
+                NSLog(@"shibboleth fragment is in the request url");
+                return YES;
+            }
+        }
+    }
+    return NO;
+}
+
++ (BOOL) isURLWithSamlFragmentByNSURLResponse:(NSURLResponse *)response {
```

Comment:
This method and the previous one are exactly the same, the only difference is the class of the parameter; and one is a subclass of the other. Let's delete one of both.